### PR TITLE
mocktikv: tiny update ScanLock and ResolveLock function

### PR DESCRIPTION
**Pull request by tiancaiamao**
Make the semantics more clear for MVCCStore, it handles raw key (tableID + PK)
it encode the key to MvccKey in the storage layer.

@disksing @MyonKeminta

### CODE REVIEW

File: store/mockstore/mocktikv/mvcc_leveldb.go [hunk 1]
```diff
@@ -800,10 +800,6 @@ func (mvcc *MVCCLevelDB) ResolveLock(startKey, endKey []byte, startTS, commitTS
 	mvcc.mu.Lock()
 	defer mvcc.mu.Unlock()
 
-	// startKey and endKey are MvccKey type actually, take out the raw key here.
```
[the remark this answers is not in the document]
Comment (tiancaiamao) [reply store/mockstore/mocktikv/mvcc_leveldb.go]: I don't remove it, just move the code here:
https://github.com/pingcap/tidb/pull/6275/files#diff-fe4f53947ee477517e64d4c2b1408e98R329

**Comment by disksing** [comment]
LGTM.

**Comment by tiancaiamao** [comment]
PTAL @MyonKeminta @coocood

File: store/mockstore/mocktikv/mvcc_leveldb.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (zhexuany) [reply store/mockstore/mocktikv/mvcc_leveldb.go]: gotcha

**Pull request merged**